Koo/Common/Settings.py

```diff
@@ -63,6 +63,7 @@ class Settings(object):
         'client.default_path': os.path.expanduser('~'),
         'client.language': False,
         'client.debug': False,
+        'client.sentry_dsn': 'sync+http://77cb0018d10842209ec638aeeffedf1a:3be0b6af34d14f7cb562b776e490ad8d@sentry.gisce.net/128',
```

Comment:
I think this not should have a default value with our sentry instance

Reply:
@ecarreras fixed